operations/research-and-development/product/release-process/release-scorecard-definitions.md

```diff
@@ -0,0 +1,237 @@
+# Release Scorecard Definitions
+
+For
+https://docs.google.com/spreadsheets/d/1Aoj4OTaWoyrKIcQNiHH1MVoRG51T20Y_0w2tg5oVw-M/edit#gid=825551144
+
+## Release Output
+
+<table>
+<colgroup>
+<col style="width: 46%" />
+<col style="width: 53%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th>Metric</th>
+<th>How to Measure</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Ratio of story-to-bug tickets</td>
+<td>Total of feature tickets over total of bug tickets. Include current quality release and previous feature release.</td>
+</tr>
+<tr class="even">
+<td>Mean time from P1 bug report to delivery</td>
+<td><p>With a new or existing Jira filter, with:</p>
+<ol type="1">
+<li>Project = Mattermost</li>
+<li>Fix Versions = Latest released version</li>
+<li>Issue Type = Bug</li>
+<li>Label = P1</li>
+<li>Created Date = 17th of the previous month</li>
+<li>Release Date = 16th of the current month</li>
+</ol>
+<p>Copy a list of Jira tickets with the above information and paste them into a spreadsheet. Calculate the average by using a formula “=Release Day-Created”.</p></td>
+</tr>
+<tr class="odd">
+<td>Mean time from P2 bug report to delivery</td>
+<td><p>With a new or existing Jira filter, with:</p>
+<ol type="1">
+<li>Project = Mattermost</li>
+<li>Fix Versions = Latest released version</li>
+<li>Issue Type = Bug</li>
+<li>Label = P2</li>
+<li>Created Date = 17th of the previous month</li>
+<li>Release Date = 16th of the current month</li>
+</ol>
+<p>Copy a list of Jira tickets with the above information and paste them into a spreadsheet. Calculate the average by using a formula “=Release Day-Created”.</p></td>
+</tr>
+<tr class="even">
+<td>Mean time from P3 bug report to delivery</td>
+<td><p>With a new or existing Jira filter, with:</p>
+<ol type="1">
+<li>Project = Mattermost</li>
+<li>Fix Versions = Latest released version</li>
+<li>Issue Type = Bug</li>
+<li>Label = P3</li>
+<li>Created Date = 17th of the previous month</li>
+<li>Release Date = 16th of the current month</li>
+</ol>
+<p>Copy a list of Jira tickets with the above information and paste them into a spreadsheet. Calculate the average by using a formula “=Release Day-Created”.</p></td>
+</tr>
+<tr class="odd">
+<td>Number of P1 bugs reported in production</td>
+<td>Bug severity guidelines are in progress, after which a Jira query will be built</td>
```

Comment:
Would we be ready to add the Jira query you use?

Reply:
Added a guideline